### PR TITLE
Add 5x13 and 6x13 ortho community layouts

### PR DESCRIPTION
**Pull request by alinelena**
## Description

Will also enable 5x13 and 6x13 ortho layouts as templates for new-keyboard.

Also fixes missing e in ortho 5x14 template.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: layouts/default/ortho_5x13/default_ortho_5x13/keymap.c [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+     * │Ctl│GUI│   │Alt│   │   │   │   │Alt│Sft│←  │ ↓ │ → │
+     * └───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┘
+     */
+    [0] = LAYOUT_ortho_5x13(
+        KC_GRV , KC_1   , KC_2  , KC_3   , KC_4  , KC_5  , KC_6  , KC_7  , KC_8   , KC_9    , KC_0   , KC_MINS, KC_BSPC,
+        KC_TAB , KC_Q   , KC_W  , KC_E   , KC_R  , KC_T  , KC_Y  , KC_U  , KC_I   , KC_O    , KC_P   , KC_LBRC, KC_RBRC,
+        KC_NUHS, KC_A   , KC_S  , KC_D   , KC_F  , KC_G  , KC_H  , KC_J  , KC_K   , KC_L    , KC_SCLN, KC_QUOT, KC_ENT ,
+        KC_LSFT, KC_NUBS, KC_Z  , KC_X   , KC_C  , KC_V  , KC_B  , KC_N  , KC_M   , KC_COMM , KC_DOT , KC_UP  , KC_SLSH,
+        KC_LCTL, KC_LGUI, KC_SPC, KC_LALT, KC_SPC, KC_SPC, KC_SPC, KC_SPC, KC_RALT, KC_RSFT , KC_LEFT, KC_DOWN, KC_RGHT
```
Comment (fauxpark) [inline layouts/default/ortho_5x13/default_ortho_5x13/keymap.c:25]:
```suggestion
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_BSPC,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,
        KC_NUHS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_UP,   KC_SLSH,
        KC_LCTL, KC_LGUI, KC_SPC,  KC_LALT, KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC,  KC_RALT, KC_RSFT, KC_LEFT, KC_DOWN, KC_RGHT
```

File: layouts/default/ortho_5x14/default_ortho_5x14/keymap.c [hunk 2]
```diff
@@ -19,7 +19,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
      */
     [0] = LAYOUT_ortho_5x14(
         KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
-        KC_ESC,  KC_TAB,  KC_Q,    KC_W,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_ESC,  KC_TAB,  KC_Q,    KC_W,    KC-E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,
```
Comment (fauxpark) [inline layouts/default/ortho_5x14/default_ortho_5x14/keymap.c:22]:
```suggestion
        KC_ESC,  KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,
```

File: layouts/default/ortho_6x13/default_ortho_6x13/keymap.c [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+     * │Ctl│GUI│   │Alt│   │   │   │   │Alt│Sft│←  │ ↓ │ → │
+     * └───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┘
+     */
+    [0] = LAYOUT_ortho_6x13(
+        KC_ESC , KC_F1  , KC_F2 , KC_F3  , KC_F4 , KC_F5 , KC_F6 , KC_F7 , KC_F8  , KC_F9   , KC_F10 , KC_DEL,  KC_BSPC,
+        KC_GRV , KC_1   , KC_2  , KC_3   , KC_4  , KC_5  , KC_6  , KC_7  , KC_8   , KC_9    , KC_0   , KC_MINS, KC_EQL ,
+        KC_TAB , KC_Q   , KC_W  , KC_E   , KC_R  , KC_T  , KC_Y  , KC_U  , KC_I   , KC_O    , KC_P   , KC_LBRC, KC_RBRC,
+        KC_NUHS, KC_A   , KC_S  , KC_D   , KC_F  , KC_G  , KC_H  , KC_J  , KC_K   , KC_L    , KC_SCLN, KC_QUOT, KC_ENT ,
+        KC_LSFT, KC_NUBS, KC_Z  , KC_X   , KC_C  , KC_V  , KC_B  , KC_N  , KC_M   , KC_COMM , KC_DOT , KC_UP  , KC_SLSH,
+        KC_LCTL, KC_LGUI, KC_SPC, KC_LALT, KC_SPC, KC_SPC, KC_SPC, KC_SPC, KC_RALT, KC_RSFT , KC_LEFT, KC_DOWN, KC_RGHT
```
Comment (fauxpark) [inline layouts/default/ortho_6x13/default_ortho_6x13/keymap.c:28]:
```suggestion
        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_DEL,  KC_BSPC,
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,
        KC_NUHS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_UP,   KC_SLSH,
        KC_LCTL, KC_LGUI, KC_SPC,  KC_LALT, KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC,  KC_RALT, KC_RSFT, KC_LEFT, KC_DOWN, KC_RGHT
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by dunk2k** [comment]
Could respective entries be added into `layouts/default/readme.md` please?

**Comment by alinelena** [comment]
> Could respective entries be added into `layouts/default/readme.md` please?

done thank you!

File: layouts/default/readme.md [hunk 4]
```diff
@@ -1433,6 +1448,23 @@ LAYOUT_ortho_5x15
...
+├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+│   │   │   │   │   │   │   │   │   │   │   │   │   │
+├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+│   │   │   │   │   │   │   │   │   │   │   │   │   │
+├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+│   │   │   │   │   │   │   │   │   │   │   │   │   │
+├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+│   │   │   │   │   │   │   │   │   │   │   │   │   │
+└───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┘
+```
```
Comment (dunk2k) [inline layouts/default/readme.md:1466]: could this diagram be moved so it appears/placed after `LAYOUT_ortho_5x`* entries?

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: layouts/default/readme.md (hunk 4 above)
Comment (alinelena) [reply layouts/default/readme.md]: done

**Comment by zvecr** [comment]
> add 5x13 and 6x13 ortho layouts in templates for new-keyboard

This is more a side effect of adding community layouts. There then is the question of if there is enough use within the repo for these to need to exist.

**Comment by alinelena** [comment]
@dunk2k of course that was all the point in addition to have them in qmk new-keyboard

**Review by dunk2k — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**